published-namespaces.txt

```diff
@@ -181,6 +181,7 @@ AndroidX.Legacy.App
 AndroidX.Legacy.Content
 AndroidX.Legacy.Widget
 AndroidX.Lifecycle
+AndroidX.Lifecycle.Viewmodel
```

Comment:
This should be capitalized as `ViewModel` for consistency with package name, etc:
- `Xamarin.AndroidX.Lifecycle.ViewModel`

Additionally, there is already a `AndroidX.Lifecycle.ViewModel` type, so we may want to rename the namespace to `ViewModels` to prevent confusion.

Reply:
Doesn't new approach handle that?

Not plural, but pascal/camel casing.